launch-web-service.sh

```diff
@@ -1,3 +1,17 @@
 #!/usr/bin/env bash
+HOST_IP=`ip addr show | egrep "e(th|np)" | grep inet | awk '{print $2}' | sed -e 's/\/[0-9]*//'`
```

Comment:
`egrep` is non-standard and deprecated. Use `grep -E` instead. (SC2196)

Reply:
Implemented the suggested code change.